packages/patrol/lib/src/binding.dart

```diff
@@ -163,10 +163,13 @@ class PatrolBinding extends IntegrationTestWidgetsFlutterBinding {
 
   @override
   void attachRootWidget(Widget rootWidget) {
-    assert(_currentDartTestFile != null);
+    assert(
+      (_currentDartTestFile != null) != (constants.hotRestartEnabled),
+      '_currentDartTestFile must be null during Hot Restart',
```

Comment:
This message doesnt say anything about the non-hot restart case. How about `'_currentDartTestFile can be null iff Hot Restart is enabled'`

Reply:
Yeah, but then the message might be to long and hard to compehend.

I'm thinking about creating two separate asserts – wdyt?

Reply:
Well its not much longer
![image](https://github.com/leancodepl/patrol/assets/29288116/494f43a5-3e2e-407a-9562-8786a4fbcfe3)

But yeah, might be hard to understand. Two separate asserts sound ok too.

Reply:
Ah, I didn't notice it's `iff`, not `if`.

I agree it's not much shorter, but at the same time it's probably too hermetic. I can see people complaining that it's a typo;

I'll just do 2 asserts. Thanks.

Reply:
I changed my mind (lol)